docs/competition/challenges/challenge.md

```diff
@@ -0,0 +1,60 @@
+---
+name: Get challenge
+
+url: https://competition.trackmania.nadeo.club
+method: GET
+route: /api/challenges/{challengeId}
+
+audience: NadeoClubServices
+
+parameters:
+  path:
+    - name: challengeId
+      type: string
+      description: A valid challenge ID
+      required: true
+---
+
+Gets challenge details for a challenge ID.
+
+---
+
+**Remarks**:
+- Note that challenges are different from competitions - challenges are separate leaderboard structures that can be part of a competition (for example in the form of a qualifying session).
+
+---
+
+**Example request**:
+```plain
+GET https://competition.trackmania.nadeo.club/api/challenges/409
+```
+
+**Example response**:
+```json
+{
+    "id": 409,
+    "uid": "8c4a18cc-dca7-4fe2-8dc3-55e40120ad26",
+    "name": "Cup of the Day 2021-08-14 #2 - Challenge",
+    "scoreDirection": "ASC",
+    "startDate": 1628989260,
+    "endDate": 1628990160,
+    "status": "SERVER_DELETED",
```

Comment:
Do we liste anywhere all of the available statuses? It could be nice to have this listed somewhere (even if it's just on 1 endpoint page) and linked to from the `Remarks` section.

Reply:
Good point, there's probably a few constants like that. Might be worth collecting them and putting them in a central place indeed.